src/libasr/ASR.asdl

```diff
@@ -317,6 +317,10 @@ ttype
     | Dict(ttype key_type, ttype value_type)
     | Pointer(ttype type)
     | CPtr()
+    | TypeVar(string var)
+
+boolop = And | Or | Xor | NEqv | Eqv
```

Comment:
You can probably remove this, we don't need it for this PR.

Reply:
I need the `| TypeVar(String var)` to for type variable declaration. I'll remove the boolop then.

Reply:
You need `TypeVar`, but you do not need the extra `boolop`.